docs/concepts/gnovm.md

```diff
@@ -24,4 +24,59 @@ This enables non-web3 developers to contribute without requiring an understandin
+`GOFMT_FLAGS`       | Change formatting of Go output. `-w` flag writes the output to the destination flag. See [Go's documentation on gofmt flags](https://pkg.go.dev/cmd/gofmt). | Uses the standard Go formatting by default.
+`GNOFMT_FLAGS`      | Change formatting of Gno output. `-w` flag writes the output to the destination flag. | Uses the standard Gno formatting by default.
+`GOIMPORTS_FLAGS`   | Adds Go formatting flags for `make imports`. | Uses values defined in `GOFMT_FLAGS` by default.
+`GOTEST_FLAGS`      | Modifies Go test suites through flags. | Prints verbose output (`-v`), defines the number of programs that can be run in parallel (`-p 1`), and times out after 30 minutes (`-timeout=30m`). 
+`GNOROOT_DIR`       | Sets the default GNOROOT. | In local development, sets GNOROOT to the supposed path of the Gno repository clone.
+`GOBUILD_FLAGS`     | Sets Go build flags in an argument list. See [Go's build flags documentation](https://pkg.go.dev/cmd/go). | Sets the arguments to pass on each Go tool link invocation to the Gno project.
+`GOTEST_COVER_PROFILE` | Defines the file where to place the cover profile. | The output location is set to `cmd-profile.out` by default/
+
+### Dev tools
+
+The dev tools section defines aliases for Go commands. See the [Makefile](https://github.com/gnolang/gno/blob/master/gnovm/Makefile) for comprehensive Go commands corresponding to the aliases.
+
+Alias command | Definition
+------------- | ----------
+`build`       | Builds Gno with `GNOBUILD_FLAGS`.
+`install`     | Installs Gno with `GNOBUILD_FLAGS`.
+`clean`       | Removes and refreshes build.
+`lint`        | Lints the golang configuration.
+`fmt`         | Formats using `GNOFMT_FLAGS`.
+`imports`     | Imports using `GOIMPORT_FLAGS`.
+
+### Test suite
+
+The test suite section defines aliases for Go commands related to tests. See the [Makefile](https://github.com/gnolang/gno/blob/master/gnovm/Makefile) for Go commands corresponding to the aliases.
+
+Alias command | Definition 
+------------- | -----------
+`test`        | Executes `_test.cmd`, `_test.pkg`, and `_test.gnolang`. 
+`_test.cmd`   | Executes tests at file location `./cmd/...` with `GOTEST_FLAGS`.
+`_test.pkg`   | Executes tests at file location `./pkg/...` with `GOTEST_FLAGS`.
+`_test.gnolang` | Executes Gnolang tests, including `native`, `stdlibs`, and `realm`. See [Makefile](https://github.com/gnolang/gno/blob/master/gnovm/Makefile) for comprehensive list.
+`test.cmd.coverage` | Runs tests on `./cmd/` and saves the output to `GOTEST_COVER_PROFILE`. 
+`test.cmd.coverage_view` | Runs `test.cmd.coverage`, then shows the results rendered in the HTML browser, and deletes the original file.
+
+### Code gen
```

Comment:
This section requires a _lot_ more explanations that a simple `generates X`, `executes Y`